index.html

```diff
@@ -2,229 +2,220 @@
-          <h2>Language</h2>
-        </div>
-
-        <h3>Javascript</h3>
-        <h3>Ruby</h3>
-        <h3>Html</h3>
-        <h3>Css</h3>
-        <div class="rectangle-container">
-          <hr class="rectangle" />
-          <h2>Frameworks</h2>
-        </div>
-        <div class="rectangle-h3">
-          <h3>bootstrap</h3>
-          <h3>Ruby and Rails</h3>
-          <h3>RSPec</h3>
-          <h3>CapyBara</h3>
-          <h3>Selenium</h3>
-        </div>
-
-        <div class="circle-container">
-          <hr class="circle" />
-          <h2>Skills</h2>
-        </div>
-        <div class="shapes-circle">
-          <h3>Codekit</h3>
-          <h3>Github</h3>
-          <h3>Codepen</h3>
-          <h3>Terminal</h3>
-        </div>
+
+      <h3>Javascript</h3>
+      <h3>Ruby</h3>
+      <h3>Html</h3>
+      <h3>Css</h3>
+      <div class="rectangle-container">
+        <hr class="rectangle" />
+        <h2>Frameworks</h2>
       </div>
-    </section>
-    <footer id="Contact-me">
-      <p>
-        I'm always interested in hearing about new projects, so if you'd like to
-        chat please get in touch.
-      </p>
-      <form action="https://formspree.io/f/meqnlzqp" method="POST">
-        <input type="text" placeholder="Full Name" maxlength="30" required />
-        <input type="text" placeholder="Email Address" required />
-        <textarea
-          name="textarea"
-          placeholder="Enter text here"
-          cols="30"
-          rows="10"
-          maxlength="500"
-          required
-        ></textarea>
-        <button class="below" type="submit">Get in touch</button>
-        <hr id="footer-divide" />
-        <ul>
-          <li><img src="./git.png" alt="" /></li>
-          <li><img src="./in.png" alt="" /></li>
-          <li><img src="./pis.png" alt="" /></li>
-          <li><img src="./T.png" alt="" /></li>
-          <li><img src="./M2.png" alt="" /></li>
-        </ul>
-
-        <hr id="footer-bottom" />
-      </form>
-    </footer>
-  </body>
+      <div class="rectangle-h3">
+        <h3>bootstrap</h3>
+        <h3>Ruby and Rails</h3>
+        <h3>RSPec</h3>
+        <h3>CapyBara</h3>
+        <h3>Selenium</h3>
+      </div>
+
+      <div class="circle-container">
+        <hr class="circle" />
+        <h2>Skills</h2>
+      </div>
+      <div class="shapes-circle">
+        <h3>Codekit</h3>
+        <h3>Github</h3>
+        <h3>Codepen</h3>
+        <h3>Terminal</h3>
+      </div>
+    </div>
+  </section>
+  <footer id="Contact-me">
+    <p>
+      I'm always interested in hearing about new projects, so if you'd like to
+      chat please get in touch.
+    </p>
+    <form action="https://formspree.io/f/meqnlzqp" method="POST">
+      <input type="text" placeholder="Full Name" maxlength="30" required />
+      <input type="text" placeholder="Email Address" required />
+      <textarea name="textarea" placeholder="Enter text here" cols="30" rows="10" maxlength="500" required></textarea>

```

Comment:
- [ ] Please add a `type=email` attribute to your email input field in order to have HTML validation 🙏 
- [ ] You could add a `margin-bottom` to each input field in order to space it out and look a bit more like the figma design 😎